src/com/pensio/api/request/ChargeSubscriptionRequest.java

```diff
@@ -4,20 +4,45 @@
 
 public class ChargeSubscriptionRequest
 {
+	/**
+	 *
+	 * @deprecated subscriptionId is deprecated, please use agreementId instead.
+	 */
 	private String subscriptionId;
+	private String agreementId;
+	private AgreementUnscheduledType agreementUnscheduledType;
 	private Amount amount;
 	private String reconciliationIdentifier;
 	
-	public ChargeSubscriptionRequest(String subscriptionId)
+	public ChargeSubscriptionRequest(String agreementId)
 	{
-		this.subscriptionId = subscriptionId;
+		this.subscriptionId = agreementId;
+		this.agreementId = agreementId;
 	}
 
+	/**
+	 *
+	 * @deprecated getSubscriptionId() is deprecated, please use {@link #getAgreementId()} instead
```

Comment:
same here, makes sense to use actual `@Deprecated` annotation.

also, it looks like the `subscriptionId` is not needed anymore, as it's private, and not used anywhere except the `getSubscriptionId()` method. maybe remove the field completely? and in the method use `agreementId` instead, like
```
/**
 * @deprecated - use getAgreementId() instead
 */
@Deprecated
public String getSubscriptionId()
{
    return agreementId;
}
```